docs/index.yml

```diff
@@ -57,13 +57,6 @@ conceptualContent:
         - url: core/tutorials/index.md
           itemType: tutorial
           text: Tutorials
-        - url: core/porting/index.md
-          itemType: how-to-guide
-          text: Port apps from .NET Framework
-      # footerLink (optional)
-      footerLink:
-        url: core/index.md
```

Comment:
wondering if we should keep this one and the next

Reply:
I did a bit of rearranging, and kept these.